core/src/components/datetime/utils/format.ts

```diff
@@ -150,11 +159,17 @@ export const generateDayAriaLabel = (locale: string, today: boolean, refParts: D
  * Gets the day of the week, month, and day
  * Used for the header in MD mode.
  */
-export const getMonthAndDay = (locale: string, refParts: DatetimeParts) => {
+export const getMonthAndDay = (locale: string, refParts: DatetimeParts, formatOptions?: Intl.DateTimeFormatOptions) => {
+  const defaultFormatOptions: Intl.DateTimeFormatOptions = { weekday: 'short', month: 'short', day: 'numeric' };
+
+  // If any options are provided, don't use any of the defaults. This way the developer can (for example) choose to not have the weekday displayed at all.
+  const options: Intl.DateTimeFormatOptions = formatOptions ?? defaultFormatOptions;
```

Comment:
~ Same feedback around not needing the explicit type here.

Reply:
thanks! fixed in b8595a36c5